SPECS/systemd/systemd-bootstrap.spec

```
@@ -37,6 +37,7 @@ Requires:       libgcrypt
 Requires:       lz4
 Requires:       pam
 Requires:       xz
+Requires(pre):  audit-libs
```

Considering our e-mail discussion:
```suggestion
Requires(post):  audit-libs
Requires(post)  pam
Requires(post)  util-linux
```